lib_test/test_httpaf.ml

```diff
@@ -154,9 +198,121 @@ module Server_connection = struct
       `Close (next_read_operation t);
   ;;
 
+  let test_synchronous_error () =
+    let request_string = "/ GET HTTP/1.1\r\n\r\n" in
```

Comment:
Does this work because it's parsing `/` as the verb and `GET` as the resource? Or did you intentionally write it in that order?

Reply:
Whoops.